### ISSUE TITLE
[Windows 11 HC] More actions menu icon is not visible until hovered in AI Android

### ISSUE THREAD

**Issue by lisli1** [microsoft/accessibility-insights-web#5614]
## Describe the bug

In Unified Canary Version 2022.609.3 with Windows 11 Desert contrast theme, the "more actions" menu button is not visible until it is hovered.

**To Reproduce**

1. Change Windows 11 contrast theme to "Desert" (from Windows settings)
2. Open Accessibility Insights for Android Canary
3. Run Automated checks on an app with accessibility issues
4. Expand an issues card
5. Note the "more actions" icon is not visible in the bottom right corner
6. Hover over the bottom right corner and see the "more actions" icon appear

## Expected behavior

"More actions" icon should be visible even before hovering over it

## Screenshots

(gif of a card in Unified Canary with Desert contrast theme, hovering and un-hovering over the "more actions" menu button with mouse cursor)
![unified-card-more-actions-button-desert-hc](https://user-images.githubusercontent.com/48259897/172970192-6dfffb5c-bd26-4b98-8955-e5a3ec1e8b2e.gif)

## Context (please complete the following information)

- OS Name & Version:  Windows 11 Version 10.0.22000 Build 22000, Desert contrast theme
- AI-Web Version & Environment:  AI Android Canary Version 2022.609.3
- Browser Version:  n/a
- Target Page:  sunflower test app, or any other app with accessibility issues

## Are you willing to submit a PR?

yes

## Did you search for similar existing issues?

yes

## Additional context

Found while verifying fix for #4616 (may be related)

**Comment by ferBonnin** [comment]
we should fix this before any release. triaged with Peter

**Comment by lisli1** [comment]
Verified the fix in Canary Version 2022.9.21.1623
